Test/Framework/Visual/VertexBufferTest.cs

```diff
@@ -121,7 +121,36 @@ public void GetPosition(bool dynamic)
                 Assert.AreEqual(savedData[2].Position, readData[2]);
                 Assert.AreEqual(savedData[3].Position, readData[3]);
             };
-            Game.RunOneFrame();
+            Game.Run();
```

Comment:
Hmm, in the XNA test project, no window at all is created for tests which call `RunOneFrame`, and because there is no window, `Draw` is never called. So all those tests, specifically those in IndexBufferTest, Texture2DTest, Texture3DTest, VertexBufferTest, result in false positives.

Tests which call `Run`, and rely on the the `ExitCondition`, do work correctly in the XNA test project - they open a window, and call `Draw` as expected.

Will keep investigating...